srcs/layouts/grek_qwerty.xml

```diff
@@ -19,9 +19,10 @@
     <key key0="φ"/>
     <key key0="γ" key2="-" key3="_"/>
     <key key0="η" key2="=" key3="+"/>
-    <key key0="ξ" key2="accent_aigu" key3="{" key4="}"/>
-    <key key0="κ" key2="accent_grave" key3="[" key4="]"/>
+    <key key0="ξ" key3="{" key4="}"/>
+    <key key0="κ" key2="" key3="[" key4="]"/>
     <key key0="λ" key2="|" key3="\\"/>
+    <key key0="accent_aigu" key1="accent_trema" key2="accent_grave"/>
```

Comment:
`key2` on the edge of the screen might be hard to type on some devices. Could it be on `key3` (bottom left) ?